ClientApp/src/components/Nav/NavMenu.js

```diff
@@ -56,11 +56,6 @@ class NavMenu extends React.Component {
                                         label="Browse"
                                         curPath={this.props.curPath}
                                     />
-                                    <StyledNavLink
```

Comment:
Yeah sorry I left this here, it was used for testing bc if you browser '/test' it refreshes the redux state.

Reply:
I don't think I explicitly deleted this, I think it's just the ghost in the proverbial git machine.